core/domain/app_feedback_report_domain.py

```diff
@@ -517,7 +519,7 @@ def require_valid_category(cls, category: str) -> None:
     @classmethod
     def require_valid_user_feedback_items_for_category(
         cls,
-        category: str,
+        category: constants.CATEGORY,
```

Comment:
While you're here, can you change the `constants` to a different name? Since `constants`` typically means a different file.

Reply:
ig, `constants.CATEGORY` point to class `CATEGORY` of `app_feedback_report_constants`. And in this file everywhere it is used as `constants.***`, why you need changes in this particular line?

Reply:
Not addressed.

Reply:
can you please suggest me any alternative name for 
`from core.domain import app_feedback_report_constants as constants`
because directly using app_feedback_report_constants 80-max length lint problem.

Reply:
Done,